### ISSUE TITLE
Conflicts, camera, renaming

### ISSUE THREAD

**Issue by brdobrdo** [CarnetApp/CarnetAndroid#107]
First install :
- setup and connection with my nextcloud
- all notes I made started to sync with nextcloud and all were conflicted
- all notifications stayed on notification page without a way to remove them or solve the conflict
- camera button doesn't respond from browser page

Reinstall:
- notes downloaded from Nextcloud automatically - OK
- camera still doesn't work from browser page
- used camera to create a note, picture taken, note renamed, after exit picture, note title and text gone, blank note
- used camera, taken picture, renamed after note was made, picture gone
- uploading, conflicted, it makes copies that are also conflicted, no way to remove conflict notifications
- browser screen gets filled with conflicted versions of notes, nextcloud folder also

Nextcloud, on my shared hosted domain (linux)
Mobile phone, Huawei P30
Carnet: installed on nextcloud and on mobile phone

Tried to remove carnet as nextcloud app to test if there is some reason for conflicts. No. Same results.

Uninstalled Carnet on Android. Will stay that way until issues get solved.

**Comment by PhieF** [comment]
Hi, on first install, did you already had notes on both nextcloud and android ?

**Comment by brdobrdo** [comment]
No. I only had one note on nextcloud. Then I installed carnet on Android and created a note there. The note from Nextcloud was synced to android but all notes made on Android become immediately conflicted. Afterwards there were several versions of the same notes reproducing back and forth.

**Comment by PhieF** [comment]
Is it possible that you have multiple nextcloud account set in the android app ?

**Comment by brdobrdo** [comment]
No. I had two before but I have removed one before I installed carnet. Should I uninstall nextcloud and install again?

**Comment by brdobrdo** [comment]
Reinstall of nextcloud and carnet. 
![Screenshot_20200520_203310_com spisoft quicknote](https://user-images.githubusercontent.com/64033592/82483819-53a31980-9ad9-11ea-8c24-caf0a43aa51f.jpg)
Just one note made.

**Comment by PhieF** [comment]
Very weird case, can you enable logs in settings, restart the app completely if it doesn't restart by itself, creae somes notes and sync them,  then send me the file in /sdcard/android/data/com.spisoft.quicknote/cache/sync.log
?

Thank you, sorry for the time it takes

**Comment by brdobrdo** [comment]
[sync.log](https://github.com/CarnetApp/CarnetAndroid/files/4658970/sync.log)
No problem. Thank you for the effort.

**Comment by PhieF** [comment]
Thanks !

If you have time, can you retry with this apk and send me logs again (it won't fix your issue, it just adds more logs)

https://cloud.exode.me/index.php/s/RLKneFFpLX2QotC

Thanks again

**Comment by brdobrdo** [comment]
Installed. Made two notes. First I think synced OK, the second not.
[sync.log](https://github.com/CarnetApp/CarnetAndroid/files/4664295/sync.log)

**Comment by PhieF** [comment]
Hi, sorry for the late reply

is it android 10 ?

thanks

**Comment by brdobrdo** [comment]
Hi, 

Yes it is. 
LP,
G



Jun 11, 2020, 11:35 by notifications@github.com:

**Comment by PhieF** [comment]
Since you've replied by mail (and I modified my answer) I'm not sure you've seen the last part:
 Did you enable logs in the settings ? Log file seems incomplete

thanks !

**Comment by brdobrdo** [comment]
[sync.log](https://github.com/CarnetApp/CarnetAndroid/files/4764969/sync.log)

**Comment by brdobrdo** [comment]
Hi

Replied on github.com 
LP,
G



Jun 11, 2020, 12:55 by notifications@github.com:

**Comment by PhieF** [comment]
Hi, sorry for the late answer...

I might have found a bug in the sync process, can you try this version one again ? 

Thanks:

https://cloud.exode.me/index.php/s/XgjDJ7d5LNMJjZg

Please try to sync multiple times

**Comment by brdobrdo** [comment]
Now it tries to sync and then stops. On nextcloud 0 b file appears.

[sync.log](https://github.com/CarnetApp/CarnetAndroid/files/4782410/sync.log)

**Comment by PhieF** [comment]
You sure sync didn't succeed ? I guess you didn't reenable logs, they seem pretty empty

**Comment by brdobrdo** [comment]
Sync didn't succeed. It shows a file in nextcloud but it is 0 B file. 

 When I check debug log in the settings and go back to settings it is unchecked. Checking it again doesn't help. 

When I delete top left note, all notes disappear. I have to exit the Latest screen and go back in to see the notes again.

When I tap camera icon in Browser screen, nothing happens.

**Comment by brdobrdo** [comment]
Also, I have some notes in Latest screen that are not visible anywhere else.

**Comment by PhieF** [comment]
It means that they were deleted but not from cache. This bug is really a tough one. And you're not the only one, I don't understand why I can't reproduce it

**Comment by brdobrdo** [comment]
Can you install an instance of nextcloud on shared hosting server and app on android? If yes, and if it works for you, could you describe the process? Maybe it is something to with the setup of nextcloud.

**Comment by brdobrdo** [comment]
One thing that makes things a bit difficult is creating a note with photo using android app. The photo is huge 9 MB. It has difficulties uploading. Maybe it would be good to resize the photo when it is taken. I don't know how software handles this during upload and sync but I can imagine if there are some edits and software tries to keep up, 9 MB doesn't help one bit.
